magellan-flow-sample/src/main/java/com/wealthfront/magellan/sample/SecondJourney.kt

```diff
@@ -0,0 +1,12 @@
+package com.wealthfront.magellan.sample
+
+import android.content.Context
+import com.wealthfront.magellan.core.Journey
+import com.wealthfront.magellan.sample.databinding.SecondJourneyBinding
+
+class SecondJourney : Journey<SecondJourneyBinding>(SecondJourneyBinding::inflate, SecondJourneyBinding::container) {
```

Comment:
I get that we want to model what a second Journey would be like, but this Journey is useless and should just be the DetailScreen inside it. We should simplify this if you're not planning to add more to it

Reply:
I am planning to show off some history rewrite functionality and flow exit stuff in the future.